config/initializers/create_background_jobs.rb

```diff
@@ -0,0 +1,5 @@
+# Create all Rufus Scheduler Jobs on Application Start
+Check.active.each do |check|
+  check.unschedule_job if check.job
```

Comment:
There is no need to unschedule a job on application startup since it can't be already scheduled.
```suggestion
```

Reply:
fixed